src/misc/cache/lv_image_header_cache.h

```diff
@@ -0,0 +1,66 @@
+/**
+* @file lv_image_header_cache.h
+*
+ */
+
+#ifndef LV_IMAGE_HEADER_CACHE_H
+#define LV_IMAGE_HEADER_CACHE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_cache_private.h"
+
+#if LV_IMAGE_HEADER_CACHE_DEF_CNT > 0
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ * GLOBAL PROTOTYPES
+ **********************/
+
+/**
+ * Initialize image header cache.
+ * @return LV_RESULT_OK: initialization succeeded, LV_RESULT_INVALID: failed.
+ */
+lv_result_t lv_image_header_cache_init(void);
+
+/**
+ * Resize image header cache.
+ * @param new_size  new size of the cache in count of image headers.
+ * @param evict_now true: evict the image headers should be removed by the eviction policy, false: wait for the next cache cleanup.
+ */
+void lv_image_header_cache_resize(uint32_t new_size, bool evict_now);
+
+#endif /*LV_IMAGE_HEADER_CACHE_DEF_CNT > 0*/
```

Comment:
How come this guard doesn't capture the last prototype in this header?

Reply:
https://github.com/lvgl/lvgl/pull/5890#pullrequestreview-1946077546 Ah, I see, please disregard.